server/src/utils/jwt.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken');
+
+const sendCookies = (payload) =>
```

Comment:
rename sendCookies to sign, send cookies is misleading in a way that you think it does sign and send cookies while it just signs it